src/components/ec-input-field/ec-input-field.vue

```diff
@@ -82,6 +94,10 @@ export default {
       default: '',
       type: String,
     },
+    tooltipLabel: {
```

Comment:
We should write a test for the new prop. We don't have to test the tooltip, but we should test the icon is there if it's set.